install/kubernetes/cilium/templates/cilium-envoy/configmap.yaml

```diff
@@ -1,5 +1,5 @@
 {{- $envoyDS := eq (include "envoyDaemonSetEnabled" .) "true" -}}
-{{- if and $envoyDS (not .Values.preflight.enabled) }}
+{{- if and $envoyDS (not .Values.preflight.enabled) (not .Values.envoy.overrideBootstrapConfigMap) }}
```

Comment:
Probably don't need this conditional; rendering this extra configmap isn't gonna hurt anything.

Reply:
I suppose not, though the idea of a configmap being created when its known to not be in use raises possibility of confusion to operators

Reply:
resolved in https://github.com/cilium/cilium/commit/9ea307db97c7c615e699c9ab47e869eef28e0c82